dependency/schema.go

```diff
@@ -20,8 +21,10 @@ type ProtoDepDependency struct {
 
 func (d *ProtoDepDependency) Repository() string {
 	tokens := strings.Split(d.Target, "/")
-	if len(tokens) > 3 {
-		return strings.Join(tokens[0:3], "/")
+	subgroupTokens := strings.Split(d.Subgroup, "/")
```

Comment:
If `Subgroup` is empty string, `strings.Split` returns `1` in Go language.
This behavior causes an error in the existing settings.

```go
subgroupTokens := make([]string, 0)
if d.Subgroup != "" {
    subgroupTokens = strings.Split(d.Subgroup, "/")
}
```